metrics-exporter-dogstatsd/src/state.rs

```diff
@@ -27,6 +27,12 @@ pub struct StateConfiguration {
 
     /// Whether or not to emit histograms as distributions.
     pub histograms_as_distributions: bool,
+
+    /// Global labels to add to all metrics
+    pub global_labels: Arc<Vec<Label>>,
```

Comment:
`StateConfiguration` isn't `Clone`, so I don't think there's any benefit to wrapping the labels in an `Arc<T>`. We can just immutably borrow them when using them with `write_counter`, etc.